docs/rst/use_cases/vulcanexus_cloud.rst

```diff
@@ -0,0 +1,150 @@
+multicast routing in cloud environments.
+
+The described scheme is represented in the following figure:
+
+.. figure:: /rst/figures/vulcanexus_cloud.png
+
+In addition to the two mentioned deployments, two |k8s| `services <https://kubernetes.io/docs/concepts/services-networking/service/>`_
+are required in order to direct dataflow to each of the pods. A LoadBalancer will forward messages reaching the cluster
+to the WAN participant of the cloud router, and a ClusterIP service will be in charge of delivering messages from the
+local DS to the listener pod. Following are the settings needed to launch these services in |k8s|:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/load-balancer-service.yaml
+    :language: yaml
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/local-service.yaml
+    :language: yaml
+
+.. note::
+
+    An `Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`_ needs to be configured for the
+    LoadBalancer service to make it externally-reachable. In this example we consider the assigned public IP address to
+    be ``2.2.2.2``.
+
+The configuration file used for the cloud router will be provided by setting up a `ConfigMap <https://kubernetes.io/docs/concepts/configuration/configmap/>`_:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/ConfigMap.yaml
+    :language: yaml
+
+Following is represented the overall configuration of our |k8s| cluster:
+
+.. figure:: /rst/figures/vulcanexus_k8s.png
+
+
+DDS-Router deployment
+---------------------
+The cloud router is launched from within a Vulcanexus Docker image, which uses as configuration file the one hosted in
+the previously set up ConfigMap. The cloud router will be deployed with the following settings:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/ddsrouter.yaml
+    :language: yaml
+
+
+Listener deployment
+-------------------
+Again, since demo nodes is not installed by default in *Vulcanexus* we have to create a new Docker image adding in this
+functionality. The Dockerfile used for the listener will slightly differ from the one utilized to launch a talker in
+our LAN, as here we need to specify the port and IP address of the local DS. This can be achieved by using the following
+Dockerfile and entrypoint:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/Dockerfile_cloud
+    :language: Dockerfile
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/run.bash
+    :language: bash
+
+As before, to build the extended Docker image it suffices to run:
+
+.. code-block:: bash
+
+    docker build -t vulcanexus-demo-nodes:galactic -f Dockerfile .
+
+Now, the listener pod can be deployed by providing the following configuration:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/listener.yaml
+    :language: yaml
+
+
+Once all these components are up and running, communication should have been established between talker and listener
```

Comment:
NIT:
Maybe add this paragraph an some other to a las subsection `Conclusion` or something like that.
I will say "you can communicate in WAN each DDS network". "Is possible to route as many times as wanted". "Dynamically add more networks connected to same Router"... Something to close up the use case :thinking: 
What do you think?

Reply:
IMHO there is not enough content as to create a new section for it, not even after including you suggestions. I would say in the future there would be more complex use cases such as this one covering what you mention.